.github/quarkus-ecosystem-issue.java

```diff
@@ -257,7 +267,7 @@ private void processITJobs(GHIssue issue, GHWorkflowJob job) {
 		} else {
 			System.out.println(String.format("Report issue found: %s - %s", issue.getTitle(), issue.getHtmlUrl().toString()));
 			System.out.println(String.format("The issue is currently %s", issue.getState().toString()));
-			Object oldIssue = issues.put(issue, job.getName().split(" / ")[0]);
+			Object oldIssue = issues.put(issue, job.getName().split(JOB_TITLE_DELIMITER)[0]);
```

Comment:
OK, so this solves the "common prefix" problem by including the delimiter for "Linux" jobs, correct?

I.e. `Q main M 24 latest windows / Mandrel build` no longer is matched by prefix `Q main M 24 latest / `, which would have been the case when the prefix previously was `Q main M 24 latest` (without the positive look behind match).

Reply:
Correct